.github/actions/prepare-tag-name/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Prepare tag name
+description: Prepare the tag name given a WordPress version number
+
+inputs:
+  tag:
+    description: Tag name for WordPress release
+    required: true
+    type: string
+outputs:
+  short:
+    description: Short tag name, eg 6.7 or 6.7.1
+    type: string
+  long:
+    description: Long tag name, eg. 6.7.0 or 6.7.1
+    type: string
+  branch:
+    description: Branch number, eg. 6.7
+    type: string
+
+runs:
+  using: composite
+  steps:
+    - name: Prepare tag name
+      shell: bash
+      run: | #shell
+        export LONG_TAG="${{ inputs.tag }}"
+        export SHORT_TAG="${LONG_TAG%.0}"
+        export BRANCH_NUMBER="${LONG_TAG%.*}"
+        echo short="$SHORT_TAG" >> "$GITHUB_OUTPUT"
+        echo long="$LONG_TAG" >> "$GITHUB_OUTPUT"
+        echo branch="$BRANCH_NUMBER" >> "$GITHUB_OUTPUT"
```

Comment:
```suggestion
          LONG_TAG="${{ inputs.tag }}"
          {
            echo "short=${LONG_TAG%.0}"
            echo "long=${LONG_TAG}"
            echo "branch=${LONG_TAG%.*}"
          } >> "$GITHUB_OUTPUT"
```

Reply:
From #17
But this does not look uniform, it is not script-like.

Reply:
What's the advantage of this?

Reply:
- no need for export
- does not look like copy-paste
- only one redirect to GITHUB_OUTPUT
- only one variable

Reply:
But it hurts the brain! 👺

Reply:
Yeah it's a bit weird isn't it. I think I'll get rid of the exports and leave it at that. I need to remove direct use of the GitHub Actions expressions too.